examples_test.go

```diff
@@ -1,21 +1,41 @@
 package scim
 
 import (
-	"log"
+	logger "log"
 	"net/http"
 )
 
 func ExampleNewServer() {
-	log.Fatal(http.ListenAndServe(":7643", Server{
+	server := Server{
 		Config:        ServiceProviderConfig{},
 		ResourceTypes: nil,
-	}))
+	}
+	logger.Fatal(http.ListenAndServe(":7643", server))
 }
 
 func ExampleNewServer_basePath() {
-	http.Handle("/scim/", http.StripPrefix("/scim", Server{
+	server := Server{
 		Config:        ServiceProviderConfig{},
 		ResourceTypes: nil,
-	}))
-	log.Fatal(http.ListenAndServe(":7643", nil))
+	}
+	// You can host the SCIM server on a custom path, make sure to strip the prefix, so only `/v2/` is left.
+	http.Handle("/scim/", http.StripPrefix("/scim", server))
+	logger.Fatal(http.ListenAndServe(":7643", nil))
+}
+
+func ExampleNewServer_logger() {
```

Comment:
I'm curious, what's the purpose of this test?

Reply:
Just an example, since it has been asked before.